blocks/hero/hero.css

```diff
@@ -18,6 +18,10 @@ main > .section-outer > .section.hero-container{
   min-height: 300px;
 }
 
+.hero.hero-picture-only {
+  padding: 0;
+}
+
 .hero h1 {
   max-width: 1200px;
```

Comment:
Can you add this shadowing to the H1? 
        `text-shadow: 1px 1px 6px #444;`

Reply:
done!